lib/config/rules/typescript.js

```diff
@@ -1,50 +1,50 @@
 // See https://github.com/nzakas/eslint-plugin-typescript
 
 module.exports = {
-  // enforces one space after the colon and zero spaces before the colon of a type annotation.
+  // Enforce one space after the colon and zero spaces before the colon of a type annotation.
   'typescript/type-annotation-spacing': ['error'],
 
   // Require explicit return types on functions and class methods
   'typescript/explicit-function-return-type': 'off',
 
-  // enforces accessibility modifiers on class properties and methods. (member-access from TSLint)
```

Comment:
I think these were usually just copied from plugin docs, I like the consistency though.